main.cpp

```diff
@@ -18,13 +18,21 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+    Rewriter& _rewriter;
+
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter){
         // Your code goes here
+
     };
 
     void run(const MatchFinder::MatchResult &Result) override {

```

Comment:
This code does not work as expected for the following example:
```cpp
// test.cpp
int main() {
    float fooooo;
    int i = (int)   fooooo   ;
    return 0;
}
```

Reply:
![image](https://user-images.githubusercontent.com/45869154/170296585-e761ea06-2ccf-4035-a568-be09d090b83b.png)
Problem fixed